src/util.clj

```diff
@@ -19,5 +21,30 @@
                  msg   (format (str/join \newline lines) ptc/the-name error)]
              (println msg))))))
 
+(defn email
+  "Email MESSAGE to TO-LIST from with SUBJECT."
+  [to-list subject message]
+  (letfn [(add-to [mail to] (.addTo mail to))
+          (add-to-list [mail to-list] (run! (partial add-to mail) to-list))]
+    (let [from (str ptc/the-name "@broadinstitute.org")]
+      (doto (-> (new SimpleEmail)
+                (.setFrom from)
+                (.setSubject subject)
+                (.setMsg message))
+        (add-to-list to-list)
+        (.setAuthentication from "fake-password")
+        (.setHostName "smtp.gmail.com")
+        (.send)))))
+
+(defn reach-out-and-touch-everyone
```

Comment:
can you name this something more descriptive? What's it doing or used for?
What does "touching everyone" mean?

Reply:
sure, that's a good idea. Will give it a more descriptive name. This is an attempt to explore ways that ptc can notify us about what it's doing through emails. It's not functional yet though.

Reply:
That reach-out-and-touch-everyone name was my idea, sorry.
It's inherited from some test code that talked to every service Zero knew about.